nbextensions/bulkImportCell/tabs/configure.js

```diff
@@ -147,16 +151,68 @@ define([
             const appSpec = specs[typesToFiles[selectedFileType].appId];
             const filePathNode = ui.getElement('input-container.file-paths');
             const paramNode = ui.getElement('input-container.params');
+            showConfigMessage();
             return Promise.all([
                 buildFilePathWidget(filePathNode, appSpec).then((instance) => {
-                    filePathWidget = instance.widget;
+                    filePathWidget = instance.instance;
```

Comment:
This was a little annoying, just a name change and now the filePathWidget and paramsWidget get stopped and removed as usual. They were being set as `undefined` before, and the stop code tests for existence first, so no errors. Yay for code coverage, I guess.